lib/google_crawler_web/views/api/success_view.ex

```diff
@@ -0,0 +1,13 @@
+defmodule GoogleCrawlerWeb.Api.SuccessView do
+  use GoogleCrawlerWeb, :view
+
+  def render("show.json", %{message: message}) do
+    %{
+      meta: [
```

Comment:
Should it be an object here instead of array 🤔

Reply:
Yea it should be object, fixed in https://github.com/ankitkalia1195/elixir-google-crawler/pull/54/commits/48ec8848c699b401bb780eb0134a25aeeed09f8a